### PR TITLE
fix: dynamic properties deprecated

### PR DESCRIPTION
**Pull request by mvarlic**
* add missing attributes for php 8.2 support

image
image
image
image
image

### CODE REVIEW

File: Model/Config/AdditionalConfigVars.php [hunk 1]
```diff
@@ -7,6 +7,7 @@ class AdditionalConfigVars implements ConfigProviderInterface
 {
     protected $getInscriptions;
     protected $configProvider;
+    protected $_getInscriptions;
```
Comment (Matiasnickolas) [inline Model/Config/AdditionalConfigVars.php]: Are needed 2 attributes with same name? `getInscriptions`  and  `_getInscriptions`

File: Model/LogHandler.php [hunk 2]
```diff
@@ -15,6 +15,12 @@ class LogHandler
     private $logFile;
     private $logDir;
     private $ecommerce;
+    private $reponse;
+    private $lockfile;
+    private $confdays;
+    private $confweight;
```
Comment (Matiasnickolas) [inline Model/LogHandler.php:21]: this should be camel case lockFile, confDays, confWeight, according to other attributes nomeclature

File: Model/Config/AdditionalConfigVars.php (hunk 1 above)
Comment (mvarlic) [reply Model/Config/AdditionalConfigVars.php]: already refactored

File: Model/LogHandler.php (hunk 2 above)
Comment (mvarlic) [reply Model/LogHandler.php:21]: that utility is replaced in another PR

**Review by Matiasnickolas — APPROVED** [review]
LGTM 💪

**Review by tribiec — APPROVED** [review]

**Pull request merged**